frontend/next.config.js

```diff
@@ -118,10 +118,6 @@ module.exports = {
     // TODO MPP-3946: Fix deprecation warnings in sass 1.80.x
     // https://github.com/mozilla/protocol/releases/tag/v18.0.0
```

Comment:
questgestion (non-blocking): we can remove these lines, right? This `TODO` is `TODONE`?
```suggestion
```

Reply:
No, there are sass warnings from mozilla-protocol, and the lingering next.js API warning.